code/controllers/subsystem/gravity.dm

```diff
@@ -0,0 +1,92 @@
+var/datum/subsystem/gravity/SSgravity
+var/global/legacy_gravity = FALSE
```

Comment:
@Cyberboss intended to be able to be switched in game if it becomes laggy (it shouldn't but just incase you know.)